instrumentation/src/main/java/io/opentelemetry/android/internal/services/Service.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android.internal.services;
+
+public interface Service extends Lifecycle {
```

Comment:
Can we default the `start()` and `stop()` methods to no-op?

Reply:
Sure, sometimes I forget we can do that... It's updated now.